### PR TITLE
Fix/cypress integration docs

### LINKED ISSUE #685: Write document about how to write integration test.
This issue just is about writing a guide in the REAMDE.md, please:

* Read the code and understand how the integration tests work, files under: `/cypress/tests/integrations/`
* Write a guide about how to create new integration and briefly describe how does this work.

---

Some hints:

- Please read our [readme](https://github.com/Greenstand/treetracker-web-map-client#treetracker-web) for more information/guide/tutorial.
- Here is [an engineering book](https://greenstand.gitbook.io/engineering/) in Greenstand.
- To know more about our organization, visit our [website](https://greenstand.org).
- If you want to join the slack community (some resources need the community member's permission), please leave your email address.
- More issues in this project: https://github.com/orgs/Greenstand/projects/26

### PR DESCRIPTION
**Pull request by keyy123**
# Description

[comment]:  Adds basic documentation on how to create an integration test in cypress and includes a basic exercise and resources. It is located at `cypress/tests/integration/readme.md`. I have also included a solution file to the basic exercise called `money_tree.cy.js`.

Fixes #685 

## Type of change: Documentation

# How Has This Been Tested?

- [X] Cypress integration

# Checklist:

- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have made corresponding changes to the documentation
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my changes
- [X] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by keyy123** [comment]
I am unsure why the package-lock.json has a merge conflict...

**Comment by dadiorchen** [comment]
@keyy123 that happens, usually I simply remove the lock file and run `npm i` locally again to generate a new one. Maybe do a rebase/merge from `master`.
I'm not sure it is the best practice

**Review by RubenSmn — COMMENTED** [review]
This looks really solid!

**Comment by keyy123** [comment]
I'll try it out @dadiorchen

File: package.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
   "name": "client",
-  "version": "2.0.0-beta.221",
+  "version": "2.0.0-beta.219",
```
Comment (dadiorchen) [inline package.json:3]: @keyy123 why you have to change the version yourself, oh, the version is wrong? we generate it automatically, I feel you operations on git is a little bit problematic, let me find a way to help you out

Comment (dadiorchen) [reply package.json:3]: I got it, it is this commit 143665f58c744bafc6c102259f21cf61b6ffd733 cause the problem, I think we can rollback this commit and try again, did you try remove the packag-lock.json, and install npm and generate it again?

**Comment by keyy123** [comment]
Yes @dadiorchen, I deleted the package-lock.json and ran npm i. I was
confused because the package-lock kept the version at 2.0.0-beta.219
instead of updating to 2.0.0-beta.221. I attempted to merge in the most
recent version of beta into the feature branch but I am unsure of the
specifics that keep reverting the repo's version.

File: package.json (hunk 1 above)
Comment (keyy123) [reply package.json:3]: yes

Comment (keyy123) [reply package.json:3]: I can try `git revert 143665f` to undo the commit history and rollback the repo to the moment before the issues happened. I think... 😅

**Comment by dadiorchen** [comment]
@keyy123 sorry for the delay, the problem is this is too big as the PR, and also, usually we shouldn't modify changelog, and package.json, so I guess there is some code/branch conflict or disorder.
I suggest to do a clean up, I idea of mine to operate the git:

1. `git checkout greenstand/beta`
2. `git checkout -b cleanup`
3. `git checkout keyy123/fix/cypress-integration-docs  -- cypress/tests/integration/money_tree.cy.js`
4. repeat above step and checkout all other valid file
5. To those file having conflict, solve the differences manually (by edit files)
6. git add . && git commit ... && git push 
7. Raise PR
```

**Comment by dadiorchen** [comment]
Hope this is helpful

**Comment by keyy123** [comment]
Ok 👌. @dadiorchen, I will fix this over the weekend!

**Comment by keyy123** [comment]
I am closing this PR b/c it has some window files from my PC that make it hard to fix. I made an alternative PR that does not have the window screenshots and I will add a loom video as an alternative.

**Pull request closed without merging**